src/applications/check-in/utils/pilotFeatures/index.js

```diff
@@ -0,0 +1,82 @@
+import wilkesBarre from './travelClaim/wilkesBarre.json';
+// import salem from './travelClaim/salem.json';
+
+const devStations = (withDev = true, withStage = true, withLocal = true) => {
+  const dev = {
+    '500': {},
+  };
+  const stage = {
+    '530': {},
+  };
+  const local = {
+    '0001': {
+      clinics: ['0001'],
+    },
+  };
+
+  return {
+    ...(withDev ? dev : {}),
+    ...(withStage ? stage : {}),
+    ...(withLocal ? local : {}),
+  };
+};
+
+const pilotFeatures = {
+  fileTravelClaim: {
+    pilotStations: {
+      ...devStations(),
+      ...wilkesBarre,
+      // Week 1 658
+      // ...salem,
+      // Week 2 565
+      // ...fayetteville,
+      // Week 3 558
+      // ...durham,
+      // Week 4 652
+      // ...hunterHolmes,
+      // Week 5 590
+      // ...hampton,
+      // Week 6 659
+      // ...salisbury,
+      // Week 7 637
+      // ...asheville,
+      // Week 8
+      // VISN 15
+    },
+  },
+  fortyFiveMinuteText: {
+    pilotStations: {
+      ...devStations(),
+    },
+  },
+};
+
+/**
+ * @param {Object} appointment
+ * @param {string} pilotFeature
+ * @returns {bool}
+ * */
+
+const isInPilot = ({ appointment, pilotFeature }) => {
+  let { clinicIen, stationNo } = appointment;
+  clinicIen = String(clinicIen);
+  stationNo = String(stationNo);
+  if (pilotFeature in pilotFeatures === false) {
+    return false;
+  }
+  const featureList = pilotFeatures[pilotFeature].pilotStations;
+  const passesClinic = () => {
```

Comment:
We could move this out and pass it `featureList` and `clinicIen`

Reply:
tell me more. Not sure I follow completely.

Reply:
Move the passesClinic method outside of the isInPilot method and pass the args it needs to it

Reply:
ah ok I get it. Is this just to remove the logic load of this function? Sorry I'm new here 😺

Reply:
I think so, and agree with wullaski on it. It would keep the code a bit more readable and testable